### PR TITLE
gsoc: wolframalpha: Add wolframalpha plugin

### PR DESCRIPTION
**Pull request by meetmangukiya**
Closes https://github.com/coala/corobo/issues/5

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: plugins/wolfram_alpha.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    """
+    Query the Computational Knowledge Engine
+    """
+
+    def activate(self):
+        super().activate()
+        self.client = wolframalpha.Client(os.environ.get('WA_TOKEN'))
+
+    @botcmd
+    def wa(self, msg, arg):
```
Comment (meetmangukiya) [inline plugins/wolfram_alpha.py]: Docstring needed

**Comment by meetmangukiya** [comment]
@Udayan12167 please reviwe

File: tests/wolfram_alpha_test.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from plugins.wolfram_alpha import WolframAlpha
+
+import vcr
+
+pytest_plugins = ['errbot.backends.test']
+extra_plugin_dir = 'plugins'
+
+my_vcr = vcr.VCR(match_on=['method', 'scheme', 'host', 'port', 'path'])
```
Comment (Udayan12167) [inline tests/wolfram_alpha_test.py]: Any reason to use vcr instead of proper html requests?

File: plugins/wolfram_alpha.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        """
+        Query the Computational Knowledge Engine.
+        """
+        res = self.client.query(arg)
+        ans = ''
+        for pod in res.pods:
+            if pod.title in ['Result', 'Results']:
+                for sub in pod.subpods:
+                    ans += sub.plaintext
```
Comment (Udayan12167) [inline plugins/wolfram_alpha.py]: Can there be multiple results? How would they get concatenated?

File: tests/wolfram_alpha_test.py (hunk 2 above)
Comment (meetmangukiya) [inline tests/wolfram_alpha_test.py]: To not make the http requests. It is not a good idea to make real requests in a test, and also, this needs an API key and encrypting and decrypting might be an Overkill

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/89/commits/4d661000e9ae58fa49ec88f25e208c1bf48f9d84

**Comment by Udayan12167** [comment]
@rultor merge

**Comment by meetmangukiya** [comment]
ack 240c364014a95887ff3e3b0511ffe803ce4282bc

**Comment by meetmangukiya** [comment]
@rultor merge

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**